internal/gcp/tags_test.go

```diff
@@ -0,0 +1,125 @@
+package gcp_test
+
+import (
+	"context"
+	"testing"
+
+	resourcemanagerpb "cloud.google.com/go/resourcemanager/apiv3/resourcemanagerpb"
+	"github.com/deliveryhero/gcp-config-connector-tagging-operator/internal/gcp/mocks"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+)
+
+func setup() (*mocks.TagsManager, context.Context) {
+	mockTagsManager := new(mocks.TagsManager)
+	ctx := context.Background()
+	return mockTagsManager, ctx
+}
+
+func TestTagsManager(t *testing.T) {
```

Comment:
These tests look odd to me - you are mocking the output for the function you are testing? Shouldn't you mock the output of the underlying GCP client, so that the code of the TagsManager funcs is tested? Maybe I'm not seeing something, but I think like this the tests add no coverage for TagsManager 🤔